### PR TITLE
[action] add environment variables for pod push

### PR DESCRIPTION
**Pull request by ghost**
### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [ ] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

**Review by mollyIV — COMMENTED** [review]
Hey @bouldering-machine 👋 

Thank you for your PR 🙇 It looks good - the naming convention seems to be consistent with what we have in other actions (i.e. `FL_CARTHAGE_USE_SSH` in `carthage` action) and these are new env variables 👍 

The only thing is could you please sign a Contributor License Agreement (CLA) according to bot comment? 🙏 Please ping me then 🙇

File: fastlane/lib/fastlane/actions/pod_push.rb [hunk 1]
```diff
@@ -65,7 +65,8 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :use_bundle_exec,
                                          description: "Use bundle exec when there is a Gemfile presented",
                                          type: Boolean,
-                                         default_value: false),
+                                         default_value: false,
+                                         env_name: "FL_POD_PUSH_USE_BUNDLE"),
           FastlaneCore::ConfigItem.new(key: :path,
```
Comment (mollyIV) [inline fastlane/lib/fastlane/actions/pod_push.rb]: Is there any reason why you didn't add an env name for a `path` parameter? 🤔

**Review by mollyIV — APPROVED** [review]
Awesome, thanks again  🎉

**Comment by joshdholtz** [comment]
@bouldering-machine It looks like the lint failed. Can you make sure you run the linter to fix this? I usually run `be rubocop -a && be rubocop -D` 😊

File: fastlane/lib/fastlane/actions/pod_push.rb [hunk 2]
```diff
@@ -65,50 +65,60 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :use_bundle_exec,
                                          description: "Use bundle exec when there is a Gemfile presented",
                                          type: Boolean,
-                                         default_value: false),
+                                         default_value: false,
+                                         env_name: "FL_POD_PUSH_USE_BUNDLE"),
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/pod_push.rb]:
```suggestion
                                         env_name: "FL_POD_PUSH_USE_BUNDLE_EXEC"),
```

File: fastlane/lib/fastlane/actions/pod_push.rb [hunk 3]
```diff
@@ -65,50 +65,60 @@ def self.available_options
...
                                        optional: true,
-                                       type: Boolean),
+                                       type: Boolean,
+                                       env_name: "FL_POD_PUSH_ALLOW_WARNINGS"),
           FastlaneCore::ConfigItem.new(key: :use_libraries,
                                        description: "Allow lint to use static libraries to install the spec",
                                        optional: true,
-                                       type: Boolean),
+                                       type: Boolean,
+                                       env_name: "FL_POD_PUSH_ALLOW_LIBRARIES"),
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/pod_push.rb]:
```suggestion
                                       env_name: "FL_POD_PUSH_USE_LIBRARIES"),
```

File: fastlane/lib/fastlane/actions/pod_push.rb [hunk 4]
```diff
@@ -65,50 +65,60 @@ def self.available_options
...
           FastlaneCore::ConfigItem.new(key: :sources,
                                        description: "The sources of repos you want the pod spec to lint with, separated by commas",
                                        optional: true,
                                        is_string: false,
                                        type: Array,
                                        verify_block: proc do |value|
                                          UI.user_error!("Sources must be an array.") unless value.kind_of?(Array)
-                                       end),
+                                       end,
+                                       env_name: "FL_POD_PUSH_REPO_SOURCES"),
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/pod_push.rb]:
```suggestion
                                       env_name: "FL_POD_PUSH_SOURCES"),
```

File: fastlane/lib/fastlane/actions/pod_push.rb [hunk 5]
```diff
@@ -65,50 +65,60 @@ def self.available_options
...
-                                       type: Boolean),
+                                       type: Boolean,
+                                       env_name: "FL_POD_PUSH_SKIP_TESTS"),
           FastlaneCore::ConfigItem.new(key: :verbose,
                                        description: "Show more debugging information",
                                        optional: true,
                                        type: Boolean,
-                                       default_value: false)
+                                       default_value: false,
+                                       env_name: "FL_POD_PUSH_SKIP_VERBOSE")
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/pod_push.rb]:
```suggestion
                                       env_name: "FL_POD_PUSH_VERBOSE")
```

**Review by joshdholtz — CHANGES_REQUESTED** [review]
@bouldering-machine I have a few suggestions on the some environment variable renames and it also looks like there is another file committed (` spaceship/lib/spaceship/tunes/build_details.rb`) that we probably want removed 🙃 Once these are addressed we can merge!

**Review by joshdholtz — CHANGES_REQUESTED** [review]
@bouldering-machine It still looks like we have two extra files in here again 😊 The other changes look good though!

Screen Shot 2019-08-13 at 7 28 35 AM

**Review by joshdholtz — APPROVED** [review]
This is a winner!! Thank you 😊❤️

**Pull request merged**